regcore/migrations/0010_auto_20160322_1704.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def forward(apps, schema_editor):
+    schema_editor.execute("""
```

Comment:
This makes sense, except that in `0009` we added the `doc_type` column and defaulted it to `'cfr'`--does that mean this `UPDATE` will hit all rows?